### PR TITLE
Set up testing framework - cucumber

### PR DESCRIPTION
**Pull request by MiriamSal**
- Added gem cucumber rails and gem database_cleaner
- Bootstrapped the application with cucumber
- Created the database and applied migration

Pvt https://www.pivotaltracker.com/story/show/160582007

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.3'
+ruby '2.5.1'
```
Comment (tochman) [inline Gemfile:2]: Why are you changing the ruby version?

File: Gemfile [hunk 2]
```diff
@@ -16,6 +16,16 @@ gem 'sass-rails', '~> 5.0'
 gem 'turbolinks'
 gem 'uglifier', '>= 1.3.0'
 
+group :development, :test do
+  gem 'cucumber-rails', require: false
+  gem 'database_cleaner'
+  gem 'factory_bot_rails'
+  gem 'pry-byebug'
+  gem 'pry-rails'
+  gem 'rspec-rails'
```
Comment (tochman) [inline Gemfile:25]: The PR title stats that this is about setting up Cucumber, but you are adding RSpec as well. Is this in scope?

**Review by tochman — COMMENTED** [review]
I'm worried about the scope of this PR

File: Gemfile (hunk 1 above)
Comment (RhonGabriel) [reply Gemfile:2]: Before starting with setting up Rspec and Cucumber, we run bundle update. The repo has a different ruby version compare to the one installed in our computer

File: Gemfile (hunk 2 above)
Comment (RhonGabriel) [reply Gemfile:25]: It is just because we had issue installing cucumber, so we did git stash drop and restarted it all.

File: Gemfile (hunk 1 above)
Comment (tochman) [reply Gemfile:2]: And is changing the ruby verion on the project a better alternative than installing 2.4.3 on your computer?

File: Gemfile (hunk 2 above)
Comment (tochman) [reply Gemfile:25]: Sorry, but that does not explain why you have commits from another PR included in this one. You should have branched off from a clean branch.

Comment (RhonGabriel) [reply Gemfile:25]: Good point. That is our mistake, we should have just created a different branch from each setup.

File: Gemfile (hunk 1 above)
Comment (RhonGabriel) [reply Gemfile:2]: Not sure if this will create a problem at later stage. But, yes, easier was just installing the ruby version stated in the code

**Comment by tochman** [comment]
Closing this one.
